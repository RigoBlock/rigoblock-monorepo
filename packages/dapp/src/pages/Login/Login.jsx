import React, { Component } from 'react'
import BaseTemplate from '../../components/templates/BaseTemplate'
import './Login.scss'

class Login extends Component {
  render() {
    return (
      <div className="Login">
        <BaseTemplate>
          <div className="login">
            <h1>Hello there!</h1>
            <p>
              Rigoblock is a decentralized app that runs on Ethereum Blockchain.
              To start using it, just follow these 3 simple steps
            </p>
          </div>
        </BaseTemplate>
      </div>
    )
  }
}

export default Login
