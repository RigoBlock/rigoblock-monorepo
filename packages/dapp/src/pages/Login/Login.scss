@import "../../components/_settings/_variables.scss";

.login {
  background: lightcoral;
  width: 50%;
  margin: 0 auto;
  h1 {
    font-size: 2 * $font-size;
  }
}
