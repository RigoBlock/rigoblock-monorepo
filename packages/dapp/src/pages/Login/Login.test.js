import { shallow } from 'enzyme'
import Login from './Login.jsx'
import React from 'react'
import toJson from 'enzyme-to-json'

describe('Login page', () => {
  it('renders correctly', () => {
    expect(toJson(shallow(<Login />))).toMatchSnapshot()
  })
})
