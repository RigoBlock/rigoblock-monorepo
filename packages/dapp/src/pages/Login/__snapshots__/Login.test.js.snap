// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login page renders correctly 1`] = `
<BaseTemplate>
  <div
    className="login-content"
  >
    <h1>
      Hello there!
    </h1>
    <p>
      Rigoblock is a decentralized app that runs on Ethereum Blockchain. To start using it, just follow these 3 simple steps:
    </p>
    <h3>
      1. Install a decentralized wallet
    </h3>
    <p>
      We suggest to use Metamask or Parity.
    </p>
    <a
      href="https://metamask.io/"
    >
      Metamask
    </a>
    <br />
    <a
      href="https://www.parity.io/"
    >
      Parity UI
    </a>
    <h3>
      2. Transfer some funds to your decentralized wallet
    </h3>
    <p>
      As you would do with a standard ETH wallet.
    </p>
    <h3>
      3. Start investing with RigoBlock
    </h3>
    <p>
      Now you are ready to deposit and invest on RigoBlock!
    </p>
    <div
      className="line-break"
    />
    <h3>
      Need a hand?
    </h3>
    <Link
      className={null}
      size="small"
      to="#"
    >
      We are here to help!
    </Link>
  </div>
</BaseTemplate>
`;
